eox_tagging/settings/common.py

```diff
@@ -18,16 +18,13 @@
 
 # Application definition
 
-INSTALLED_APPS = [
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'rest_framework',
-    'eox_tagging',
-]
+INSTALLED_APPS = []
```

Comment:
If you remove this I don't think the migrations will run properly, please check that